app/clients/email/aws_ses.py

```diff
@@ -61,40 +61,69 @@ def init_app(self, region, statsd_client, *args, **kwargs):
     def get_name(self):
         return self.name
 
-    def send_email(self,
-                   source,
-                   to_addresses,
-                   subject,
-                   body,
-                   html_body='',
-                   reply_to_address=None,
-                   attachments=[]):
-        try:
-            if isinstance(to_addresses, str):
-                to_addresses = [to_addresses]
-
-            source = unidecode(source)
-
-            reply_to_addresses = [reply_to_address] if reply_to_address else []
-
-            multipart_content_subtype = 'alternative' if html_body else 'mixed'
-            msg = MIMEMultipart(multipart_content_subtype)
+    def send_email(
+        self,
+        source,
+        to_addresses,
+        subject,
+        body,
+        html_body='',
+        reply_to_address=None,
+        attachments=[],
```

Comment:
I suggest you not do mutable default arguments
Here's some context: https://docs.python-guide.org/writing/gotchas/

Reply:
Thanks for sharing that. Though I'm not the author of this code and we don't rely on this at the moment, I've changed that in the next commit.